### DISCUSSION TITLE
Copy an object to another photo

### DISCUSSION THREAD

**Discussion by luisamanett** [lllyasviel/Fooocus#2475] [Q&A]
Hi everyone, thank you for this fabulous project, I wanted to know if anyone can help me, I create handmade hats, I created a girl and I wanted to make the model wear my hat, I put the model in inpaint and select the head area , and I put the photo of my hat in image prompt but I don't know how to get the girl to wearmy hat exactly, it changes my style a lot, what settings and styles could I try? Thank you.

**Comment by mashb1t** [comment]
I'd suggest to use inpainting with a mask around your hat (https://github.com/lllyasviel/Fooocus/pull/2032 may help to automate this) and the person as image prompt + enable inpaint and image prompt mixing in developer debug mode.

**Reply by luisamanett** [reply]
I'll try, thank you very much
